lib/gengo-ruby/api_handler.rb

```diff
@@ -475,13 +486,15 @@ def getGlossaryList(params = {})
 
     # Gets one glossary that belongs to the authenticated user
     def getGlossary(params = {})
-      self.get_from_gengo('translate/glossary/:id'.gsub(':id', params.delete(:id).to_s), params)
+      id = params.delete(:id).to_s
+      self.get_from_gengo("translate/glossary/#{id}", params)
     end
 
     # Downloads one glossary that belongs to the authenticated user
     def getGlossaryFile(params = {})
```

Comment:
I checked all modifications I wrote on this PR except this function.
It doesn't seem like work. If already gengo stopped to provide this API endpoint then I think you want to remote this function.